src/sage/combinat/independence_posets.py

```diff
@@ -0,0 +1,845 @@
+"""
```

Comment:
This file could be linked in the page https://preview-37601--sagemath.netlify.app/html/en/reference/combinat/sage/combinat/posets/all#sage-combinat-posets-all